mpp/build.gradle.kts

```diff
@@ -110,21 +110,8 @@ tasks.register("publishComposeJbExtendedIconsToMavenLocal", ComposePublishingTas
     iconsPublications()
 }
 
-tasks.register("testDesktop") {
-    dependsOn(":compose:desktop:desktop:jvmTest")
-    dependsOn(":compose:animation:animation:desktopTest")
-    dependsOn(":compose:animation:animation-core:desktopTest")
-    dependsOn(":compose:ui:ui:desktopTest")
-    dependsOn(":compose:ui:ui-graphics:desktopTest")
-    dependsOn(":compose:ui:ui-text:desktopTest")
-    dependsOn(":compose:ui:ui-test-junit4:desktopTest")
-    dependsOn(":compose:foundation:foundation:desktopTest")
-    dependsOn(":compose:foundation:foundation-layout:desktopTest")
-    dependsOn(":compose:material:material:desktopTest")
-    dependsOn(":compose:material:material-ripple:desktopTest")
-    dependsOn(":compose:runtime:runtime:desktopTest")
-    dependsOn(":compose:runtime:runtime-saveable:desktopTest")
-    dependsOn(":compose:material3:material3:desktopTest")
+tasks.register("desktopCICheck") {
```

Comment:
Let's keep naming `testDesktop` may be? I don't see any reasons to change it here...

Reply:
It is on purpose named this way:
1. `check`, because `test` run only tests, and check run tests, API validation, and something else in the future.
2. we can't reuse standard `check` - android doesn't compile, and we need separate tasks for different CI configurations. Also we can't reuse `desktopCheck` - KMP doesn't have it (but it seems it should be)
3. I specifically mention CI, to tell that it this task runs on CI, and if we need to add anything, we should add here, not on CI itself. But now I am not sure, if it is needed.
4. `check` in the end instead of the beginning, to be consistent with KMP Gradle task naming style - desktopTest, desktopJar, ... The style itself is inconsistent though (there is compileKotlinDesktop), so we can be consistent with our current tasks instead 🤔.

So, the variants:
- checkDesktop
- desktopCheck
- desktopCICheck

renamed to the first variant.